app/main.py

```diff
@@ -1 +1,71 @@
-# write your code here
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error messages for `NotVaccinatedError` and `OutdatedVaccineError` should be distinct to accurately reflect the specific issue. Consider using different messages for each exception to provide clearer feedback.